app/assets/javascripts/admin/tables.js

```diff
@@ -10,6 +10,8 @@ $(document).on('turbolinks:load',function(){
       table = $(tableElementId).DataTable();
```

Comment:
The only issue I see with this PR is with the back button again.
When going back, the table has again the header with the number of entries to show and the search bar. Adding the hash option here as well, partially solves the problem, as the table is now ok but narrower for some reason. Not sure wether this would be the best solution though